### DISCUSSION TITLE
Any plan to support Pandas ^2.0.0?

### DISCUSSION THREAD

**Discussion by rbreply** [etna-team/etna#399] [Q&A] [answered]
Hi all! 
Is there any plan to maintain Etna with the new pandas?

**Comment by d-a-bunin** [comment, marked as answer]
Hi!
- Currently, we have a dependency `pytorch-forecasting^0.9.0` which stops us from working with `pandas^2`.
- We are going to get rid of `pytorch-forecasting^0.9.0` in a next major release, so it will be possible to allow `pandas^2`.

**Reply by callmeBalloch** [reply]
it looks like 2w ago a new release of `pytorch-forecasting^1.1.0` was published https://github.com/sktime/pytorch-forecasting/releases is this something can help with pandas 2?

**Reply by d-a-bunin** [reply]
The possible plan is to abandon `pytorch-forecasting` in next major release of `etna`, because we now have native implementations of TFT and DeepAR.

**Reply by callmeBalloch** [reply]
thanks @d-a-bunin I see a lot of progress on this, thanks again for the efforts! 

After the latest releases of pytorch-forecasting there might be still the chance to use it maybe with an additional extra=['pytorch-forecasting']. But if you plan to keep only the Native ones it's super! Any plan for the mayor release?

**Comment by egoriyaa** [comment]
Solved in #599
